easybuild/easyconfigs/m/mDust/mDust-20231029-GCC-10.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'mDust'
+version = '20231029'
+
+homepage = 'https://github.com/lh3/mdust'
+description = "mdust from DFCI Gene Indices Software Tools (archived for a historical record only)"
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+sources = [{
+    'filename': 'mdust.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/lh3',
+        'repo_name': 'mdust',
+        'commit': '3e3fed8',
+        'clone_into': 'mdustSrc',
```

Comment:
The default is to use the name of the repo (`mdust`), which is fine I guess, so I'd leave this out:
```suggestion
```